### PR TITLE
ft: ZENKO-1755 Add hdclient

### PR DESCRIPTION
**Pull request by benoit-a**
## Description

Add hdclient support

### Motivation and context

Hyperdrive is a new storage system by Scality. hdclient is a JS module to access it.

This PR adds "hdclient" as new possible backend for storage.

### Related issues

ZENKO-1755

### CODE REVIEW

File: lib/utilities/reportHandler.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ function getCapabilities() {
         locationTypeSproxyd: true,
         locationTypeNFS: true,
         locationTypeCephRadosGW: true,
+        locationTypeHyperdrive: true,
```
Comment (rachedbenmustapha) [inline lib/utilities/reportHandler.js]: We may have over-engineered this but as practice for incomatible format upgrades, we went ahead with full-v2 breakage :) So the Orbit stack will check for the `locationTypeHyperdriveV2` capability in order to show the new version of the hyperdrive settings.

Comment (benoit-a) [reply lib/utilities/reportHandler.js]: thx, will fix

**Review by rachedbenmustapha — APPROVED** [review]
LGTM, provided the hdclient backend itself honors versioning requirements as the location constraint advertises

**Review by rahulreddy — APPROVED** [review]

**Review by fferrandis — APPROVED** [review]

**Comment by benoit-a** [comment]
@bert-e approve

**Pull request merged**